src/test/java/com/karankumar/bookproject/ui/goal/ReadingGoalViewTest.java

```diff
@@ -205,7 +207,7 @@ private Book createBook(ShelfName shelfName) {
     }
 
     @Test
-    void correctInformationShownWhenGoalIsSetOrUpdated() {
+    void correctInformationWhenGoalIsSetOrUpdated() {
```

Comment:
Could you add `show` to the beginning of this method? Otherwise, the sentence does not make sense: `ReadingGoalView should correctInformationWhenGoalIsSetOrUpdated...`?